GitUI/CommandsDialogs/FormDiff.cs

```diff
@@ -82,6 +86,13 @@ private void PopulateDiffFiles()
             DiffFiles.SetDiffs(ckCompareToMergeBase.Checked
                 ? new[] { _headRevision, _mergeBase }
                 : new[] { _headRevision, _baseRevision });
+
+            // Bug in git-for-windows: Comparing working directory to any branch, fails, due to -R
+            // I.e., git difftool --gui --no-prompt --dir-diff -R HEAD fails, but
+            // git difftool --gui --no-prompt --dir-diff HEAD succeeds
+            // Thus, we disable comparing "from" working directory.
```

Comment:
Open with difftool button is still enabled

Reply:
Yes - the issue was `            openWithDifftoolToolStripMenuItem.Enabled = enableDifftoolDirDiff;` rather than `            btnCompareDirectoriesWithDiffTool.Enabled = enableDifftoolDirDiff;`. A bug.